src/inspect_ai/tool/_tools/_computer/_resources/Dockerfile

```diff
@@ -0,0 +1,103 @@
+FROM docker.io/ubuntu:22.04
```

Comment:
Let's see if we can get this integrated with our internal image building scheme (should be quite easy).